CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+## Contributing
+
+First off, thanks for considering contributing to Autoland! It's people like you that make this project a great tool.
```

Comment:
Needs to be changed from Autoland to GeoFS Plugins

Reply:
```suggestion
First off, thanks for considering contributing to this project! It's people like you that make this project a great tool.
```
How does that look?